StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Views/PayWithLinkButton.swift

```diff
@@ -109,13 +109,25 @@ final class PayWithLinkButton: UIControl {
         let payWithLinkString = NSMutableAttributedString(string: String.Localized.pay_with_link)
 
         // Create the Link logo attachment
-        let linkImage = Image.link_logo.makeImage(template: true)
+        let linkImage = LinkUI.useNewBrand ? Image.link_logo_bw.makeImage(template: false) : Image.link_logo_deprecated.makeImage(template: true)
         let linkAttachment = NSTextAttachment(image: linkImage)
-        let linkLogoRatio = linkImage.size.width / linkImage.size.height
 
-        // Link logo should be a little taller than the height of a cap in SF Medium
-        let linkLogoHeight = linkView.font.capHeight + (linkView.font.pointSize * 0.1)
-        linkAttachment.bounds = CGRect(x: 0, y: 0, width: linkLogoHeight * linkLogoRatio, height: linkLogoHeight)
+        if LinkUI.useNewBrand {
+            let linkLogoRatio = linkImage.size.width / linkImage.size.height
+
+            let linkTextSpacing = 0.073 // the total top+bottom space outside the Link logo
```

Comment:
Looks like these two lines are shared between the if/else and can be pulled out before the if statement?